Second commit

seamarine opened the pull request.

Commit:
Second commit

alesyaisachenko commented on src/main/java/pageObject/MainPage.java:
```diff
@@ -0,0 +1,89 @@
+    public static final String URL = "https://qa-scooter.praktikum-services.ru/";
+
+    // Открыть ссылку
+    public MainPage open() {
```
Это нужно обязательно исправить:

В классе вначале идет объявление все переменных, потом идут методы.

Подправь, плиз, во всем коде.

alesyaisachenko commented on src/main/java/pageObject/MainPage.java:
```diff
@@ -0,0 +1,89 @@
+        return driver.findElement(accordionText(number)).getText();
+    }
+
+    private final By orderButton_1 = By.className("Button_Button__ra12g");
```
Это нужно обязательно исправить:

orderButton_1  -- не удачный нейминг.

1. В имени не должно быть символа "_".
2. Плохая практика использовать порядковые номера в названиях.

alesyaisachenko commented on src/main/java/pageObject/OrderDetails.java:
```diff
@@ -0,0 +1,157 @@
+    }
+
+    // Кнопка Заказать сверху
+    private final By buttonOrderInHeader = By.xpath("//button[@class='Button_Button__ra12g']");
```
Это нужно обязательно исправить:

этот элемент лежит в MainPage

alesyaisachenko commented on src/main/java/pageObject/OrderDetails.java:
```diff
@@ -0,0 +1,157 @@
+    private final By buttonOrderInHeader = By.xpath("//button[@class='Button_Button__ra12g']");
+
+    // Кнопка Заказать снизу
+    private final By buttonOrderDown = By.xpath(".//button[@class='Button_Button__ra12g Button_Middle__1CSJM']");
```
Это нужно обязательно исправить:

этот элемент лежит в MainPage

alesyaisachenko commented on src/main/java/pageObject/OrderDetails.java:
```diff
@@ -0,0 +1,157 @@
+            // (button.equals("Нижняя кнопка"))
+            driver.findElement(buttonOrderInHeader).click();
+        }
+        if (button.equals("Нижняя кнопка")) {
```
Это нужно обязательно исправить:

в двух if проверяется одно и то же условие

Commit:
Second commit:
- удалила дублирующиеся переменные кнопок Заказать;
- поправила if;
- поправила расположение переменных и методов.

seamarine commented:
Надеюсь, все правильно сделала и исправления нужно делать в самом первом рул реквесте. Спасибо за замечания)
